libs/fn/src/lib/notification/components/notification-container/notification-container.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, ContentChild } from '@angular/core';
+import { NotificationActionButtonsDirective } from '../../directives/notification-action-buttons.directive';
+import { NotificationDismissButtonDirective } from '../../directives/notification-dismiss-button.directive';
+import { NotificationSemanticIconDirective } from '../../directives/notification-semantic-icon.directive';
+import { NotificationSemanticTitleDirective } from '../../directives/notification-semantic-title.directive';
+import { NotificationTextDirective } from '../../directives/notification-text.directive';
+import { NotificationTitleDirective } from '../../directives/notification-title.directive';
+
+@Component({
+    selector: 'fn-notification-container',
+    templateUrl: './notification-container.component.html'
+})
+export class NotificationContainerComponent {
```

Comment:
Tree-shaking is questionable here, since user may not use most of these directives when used. Consider alternative approach. MINOR note